Sources/TokamakDemo/TokamakDemo.swift

```diff
@@ -17,52 +17,122 @@
 
 import TokamakShim
 
+func title<V>(_ view: V, title: String) -> AnyView where V: View {
+  if #available(OSX 10.16, iOS 14.0, *) {
+    return AnyView(view.navigationTitle(title))
+  } else {
+    #if !os(macOS)
+    return AnyView(view.navigationBarTitle(title))
+    #else
+    return AnyView(view)
+    #endif
+  }
```

Comment:
Turns out there's a way to avoid `AnyView`:

```suggestion
@ViewBuilder
func title<V>(_ view: V, title: String) -> some View where V: View {
  if #available(OSX 10.16, iOS 14.0, *) {
    view.navigationTitle(title)
  } else {
    #if !os(macOS)
    view.navigationBarTitle(title)
    #else
    view
    #endif
  }
  }
```